src/adjustablearrowcap.c

```diff
@@ -169,6 +169,11 @@ GdipCreateAdjustableArrowCap (REAL height, REAL width, BOOL isFilled, GpAdjustab
 	cap->width = width;
 	cap->height = height;
 
+	if (height == 0)
```

Comment:
shouldn't it check for width == 0 instead? otherwise we could have a divide by zero below.

Reply:
Thanks! This is totally a bug, my bad. I added some more edge-case tests that would have failed without this.